docker/Dockerfile

```diff
@@ -25,6 +24,8 @@ RUN mkdir -p /opensearch-benchmark/.benchmark && \
 
 USER 1000
 
+ARG BUILD_DATE
```

Comment:
Have we updated the relevant docker build command to pass this argument? @IanHoang

Reply:
CI Jenkins build already passes this in but it's not necessary for users who want to experiment with the dockerfile since it just affects the label. I can add it to the documentation still though.

Reply:
Can you please share the CI job that will build this dockerfile?

Reply:
CI Job: https://build.ci.opensearch.org/job/opensearch-benchmark-release/
Previous Example of Docker Build: https://build.ci.opensearch.org/job/opensearch-benchmark-release/6/console.

Downloaded the previous Dockerhub release and it has the BUILD_DATE and VERSION already incorporated:
```
            "Labels": {
                "org.label-schema.build-date": "2023-04-28T00:47:36Z",
                "org.label-schema.description": "A community driven, open source project to run performance tests for OpenSearch",
                "org.label-schema.license": "Apache-2.0",
                "org.label-schema.name": "opensearch-benchmark",
                "org.label-schema.schema-version": "1.0",
                "org.label-schema.url": "https://opensearch.org/",
                "org.label-schema.vcs-url": "https://github.com/opensearch-project/OpenSearch-Benchmark",
                "org.label-schema.vendor": "OpenSearch-Project",
                "org.label-schema.version": "0.4.1"
            }
        },
```